src/UI/Implementation/Component/Input/ViewControl/Group.php

```diff
@@ -65,8 +65,12 @@ public function withNameFrom(NameSource $source, ?string $parent_name = null): s
      */
     public function getContent(): Result
     {
-        if (empty($this->getInputs())) {
-            return new Ok([]);
+        $inputs = array_filter(
```

Comment:
Hi @nhaagen,

I think this will bite us at some point, since this adds some element of surprise. I might not grasp the complete problem, so take this with a grain of salt. But I think we would be better of if the `NullControl` has `OK(null)` content (if it does not already have that) and we move that special case for filtering to the usage location where we actually need it.

Maybe it would be good to discuss this.

Kind regards!

Reply:
Hi @klees ,
you're probably right, the filtering is too radical in the group.
I reverted the changes in Group and added the null-result to the ViewControl..